src/Library/demos/Switch/code.rs

```diff
@@ -0,0 +1,31 @@
+use crate::workbench;
+use gtk::prelude::*;
+
+pub fn main() {
+    let switch_on: gtk::Switch = workbench::builder().object("switch_on").unwrap();
+    let label_on: gtk::Label = workbench::builder().object("label_on").unwrap();
+
+    let switch_off: gtk::Switch = workbench::builder().object("switch_off").unwrap();
+    let label_off: gtk::Label = workbench::builder().object("label_off").unwrap();
+
+    let switch_on_clone = switch_on.clone();
+    let switch_off_clone = switch_off.clone();
```

Comment:
Make weak references with `glib::clone` instead.
This should solve your lifetime issues.

You can read more about this here: https://gtk-rs.org/gtk4-rs/stable/latest/book/g_object_memory_management.html

Reply:
Changed, please check now.